src/api/v1beta1/properties.go

```diff
@@ -359,6 +364,11 @@ func (dk *DynaKube) HostGroup() string {
 	return hostGroup
 }
 
+// UseActiveGateAuthToken returns if the activeGate should get an authToken mounted
```

Comment:
```suggestion
// UseActiveGateAuthToken returns if the activeGate should get an authToken mounted
// By default its should be used if the `dk.spec.apiToken` has the correct scopes,
// but otherwise the user can also opted out using a feature-flag
```

Reply:
(writing this comment made me write the little essay above 😛 )

Reply:
it's an essay already, so actual scope- and feature-flag- names can be mentioned here explicitly